listenbrainz/webserver/static/js/src/SpotifyPlayer.tsx

```diff
@@ -141,7 +141,6 @@ export default class SpotifyPlayer extends React.Component<
         }
         this.handleWarning("Could not find track on Spotify");
         this.playNextTrack();
-        return;
```

Comment:
Extra return statements, removed by the linter. I checked if player was working as expected and the alert bug didn't appear. This should be safe to remove.